orchestrator/api/system.py

```
@@ -1085,6 +1085,19 @@ async def get_system_state_summary(ctx: RequestContext = Depends(get_request_con
     try:
+        # PRD-166 S2: real field-memory health — ping Qdrant instead of a
+        # hardcoded 'healthy', so an outage is actually reported.
+        field_theory_status = "unknown"
+        try:
+            from modules.context.factory import get_shared_context
+            _field = get_shared_context()
+            _inner = getattr(_field, "_inner", _field) if _field else None
+            if _inner is not None and hasattr(_inner, "health"):
+                _h = await _inner.health()
+                field_theory_status = "healthy" if _h.get("healthy") else "unhealthy"
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)